arkane/statmech.py

```diff
@@ -1175,6 +1175,32 @@ def determine_rotor_symmetry(energies, label, pivots):
                      ' based on the {3}.'.format(symmetry, label, pivots, reason))
     return symmetry
 
+def is_linear(coordinates):
```

Comment:
We already had the `is_linear` function in Arkane. I think it got deleted somewhere in this PR, and from briefly looking at it the added function seems extremely similar if not identical. Please change the PR so the function doesn't get deleted and re-added

Reply:
We did, it got deleted in a prior PR and I had to recreate it after I rebased.

Reply:
(It's deleted on master)

Reply:
Can you double-check? I see it on mester here:
https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/arkane/statmech.py#L737

Reply:
I was incorrect, but I've squashed them so it shouldn't show up as deleted and created in a different commit